.gitignore

```diff
@@ -1,5 +1,13 @@
 _build/
 _site/
+Properties/
+node_modules/
+wwwroot/
+
+#override .sln when required. Not needed in most cases.
+*.sln
+
+Project_Readme.html
```

Comment:
Better note would be good, but before we go that route I'd like to get Shayne's take on it, as I'm still not convinced it's the best choice.